src/js/components/FormField/__tests__/__snapshots__/FormField-test.tsx.snap

```diff
@@ -161,8 +161,8 @@ exports[`FormField Field with null as child 1`] = `
 .c3 {
   margin-left: 0px;
   margin-right: 0px;
-  margin-top: 6px;
-  margin-bottom: 6px;
+  margin-top: 12px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below